### DISCUSSION TITLE
[V5] How to add index to timestamps

### DISCUSSION THREAD

**Discussion by Pratikkshirsagar** [adonisjs/core#2635] [Help] [answered]
Hi,
I'm trying to index ```created_at``` & ```updated_at``` column using ```timestamps``` in migration

```migration file```
```ts 
 public async up(): Promise<void> {
    this.schema.createTable(this.tableName, (table) => {
      table.increments('id').primary()
      table.string('email').unique().notNullable()
      table.string('password').notNullable()
      table.timestamps(true).index();
    });
  }
```
while running the migration it is throwing error of ```Cannot read property 'index' of undefined```
How can I index ```timestamps```?

**Comment by aayush123** [comment]
You can simply add `table.index('created_at')` & `table.index('updated_at')` after `table.timestamps(true)`

**Comment by thetutlage** [comment, marked as answer]
I recommend using the `timestamp` and not `timestamps`. I updated the docs that explains why NOT use `timestamps`. https://docs.adonisjs.com/reference/database/table-builder#timestamps
